src/main/java/users/controllers/UsersController.java

```diff
@@ -0,0 +1,11 @@
+package users.controllers;
+
+import static spark.Spark.get;
+
+public class UsersController {
+
+  public static void greetUser() {
```

Comment:
Static method. Nice touch.
Just for you to know, you can define the gets inside `UsersController` constructor.

```
public class UserController {
    public UserController() {
        get("/hello/:name ", (request, response) -> "Hello " + request.params(":name"));
    }
}
```

Then, in the main class you just have to instantiate the controller

```
new UsersController()
```

Voila!